### ISSUE TITLE
`argname.definition` is not a function

### ISSUE THREAD

**Issue by ming-codes** [fabiosantoscode/terser#80]
**Bug report**

**Version (complete output of `terser -V`)**

The bug is introduced in `3.7.7`. `3.7.6` works fine.

**Complete CLI command or `minify()` options used**

```
> terser --compress --mangle -- bad-syntax.js 
```

**`terser` input**

```js
export default function(data = []) {
  if (arguments.length === 1) {
    data = this.get('content');
  }

  return data;
};
```

**`terser` output or error**

```js
ERROR: argname.definition is not a function
    at eval (eval at <anonymous> (/Users/mingliu/.config/yarn/global/node_modules/terser/tools/node.js:21:1), <anonymous>:10837:39)
    at Array.forEach (<anonymous>)
    at TreeWalker.eval [as visit] (eval at <anonymous> (/Users/mingliu/.config/yarn/global/node_modules/terser/tools/node.js:21:1), <anonymous>:10835:31)
    at TreeWalker._visit (eval at <anonymous> (/Users/mingliu/.config/yarn/global/node_modules/terser/tools/node.js:21:1), <anonymous>:1545:24)
    at AST_Function._walk (eval at <anonymous> (/Users/mingliu/.config/yarn/global/node_modules/terser/tools/node.js:21:1), <anonymous>:769:24)
    at AST_Function.walk (eval at <anonymous> (/Users/mingliu/.config/yarn/global/node_modules/terser/tools/node.js:21:1), <anonymous>:467:21)
    at AST_Function.eval [as drop_unused] (eval at <anonymous> (/Users/mingliu/.config/yarn/global/node_modules/terser/tools/node.js:21:1), <anonymous>:10902:14)
    at Compressor.before (eval at <anonymous> (/Users/mingliu/.config/yarn/global/node_modules/terser/tools/node.js:21:1), <anonymous>:7787:17)
    at AST_Function.eval [as transform] (eval at <anonymous> (/Users/mingliu/.config/yarn/global/node_modules/terser/tools/node.js:21:1), <anonymous>:4650:35)
    at eval (eval at <anonymous> (/Users/mingliu/.config/yarn/global/node_modules/terser/tools/node.js:21:1), <anonymous>:4850:76)
```

**Expected result**

No error

**Comment by kzc** [comment]
Thanks for the bug repro. Fix in the works.

**Comment by fabiosantoscode** [comment]
Released as of v3.7.8
